Cargo.toml

```diff
@@ -287,7 +287,7 @@ postgres-openssl = { version = "0.5.0", default-features = false, features = ["r
 pulsar = { version = "4.1.3", default-features = false, features = ["tokio-runtime", "auth-oauth2"], optional = true }
 rand = { version = "0.8.5", default-features = false, features = ["small_rng"] }
 rand_distr = { version = "0.4.3", default-features = false }
-rdkafka = { version = "0.28.0", default-features = false, features = ["tokio", "libz", "ssl", "zstd"], optional = true }
+rdkafka = { git = "https://github.com/fede1024/rust-rdkafka", tag = "v0.29.0", default-features = false, features = ["tokio", "libz", "ssl", "zstd"], optional = true }
```

Comment:
It's odd that they tagged it but didn't release it to crates.io 🤔

Reply:
Left a comment over here asking if that was intentional: https://github.com/fede1024/rust-rdkafka/issues/478#issuecomment-1275425106